lib/ansible/utils/hashing.py

```diff
@@ -23,10 +23,7 @@
 
 # Note, sha1 is the only hash algorithm compatible with python2.4 and with
 # FIPS-140 mode (as of 11-2014)
```

Comment:
Feels like this comment should be updated or removed. Python 2.4 hasn't been supported on the controller or targets since 2017.